### DISCUSSION TITLE
Provide Debian 11 aarch64 packages

### DISCUSSION THREAD

**Discussion by bastelfreak** [puppetlabs/pdk#1330] [Feature Request] [answered]
## Use Case

As a developer I would like to install pdk on my ARM workstation. Please provide packages for Debian 11 aarch64.

## Describe the Solution You Would Like

Please provide packages for Debian 11 aarch64.

## Additional Context

ARM boards are now available for the mainstream, as workstation and also as server.

**Comment by david22swan** [comment]
We are currently looking into expanding support for additional architectures and so there should be an answer for this soon.

**Comment by bastelfreak** [comment]
is that a discussion that's public anywhere or something where partners and customers can provide feedback?

**Reply by david22swan** [reply]
To clarify, we have decided to go forward with it, the investigation is into the amount of work required and how soon we can accomplish it.
Which does not have an answer at this time.

**Reply by bastelfreak** [reply]
ahhhh okay. Well those are great news!

**Reply by david22swan** [reply, marked as answer]
As a followup to this, just to keep you informed, we're having to cut an immediate release for the PDK prior to this getting done, due to an out of date certificate in the last one.
So it won't be in the next release as was planned.
The work will continue however and if everything goes to plan it will be in the next.
